big_tests/tests/login_SUITE.erl

```diff
@@ -38,17 +38,26 @@ all() ->
      {group, login_scram_store_plain},
      {group, login_specific_scram},
      {group, login_scram_tls},
+     {group, fast_tls},
      {group, messages},
      {group, access}
     ].
 
 groups() ->
     [{login, [parallel], all_tests()},
      {login_digest, [sequence], digest_tests()},
+     %% The SCRAM tests below have SCRAM_PLUS tests skipped
+     %% because SCRAM_PLUS is not implemented for just_tls yet
      {login_scram, [parallel], scram_tests()},
      {login_scram_store_plain, [parallel], scram_tests()},
      {login_scram_tls, [parallel], scram_tests()},
      {login_specific_scram, [sequence], configure_specific_scram_test()},
+     %% rerun SCRAM tests with fast_tls including SCRAM_PLUS tests
+     %%{fast_tls, [{group, login_specific_scram}]},
```

Comment:
Is this commented out line intentional?

Reply:
It's a leftover from dev testing, will remove.